ctms/models.py

```diff
@@ -15,52 +16,58 @@
 from sqlalchemy.dialects.postgresql import UUID
 from sqlalchemy.ext.declarative import declared_attr
 from sqlalchemy.ext.hybrid import Comparator, hybrid_property
-from sqlalchemy.orm import relationship
+from sqlalchemy.orm import DeclarativeBase, Mapped, mapped_column, relationship
 from sqlalchemy.sql.functions import func
 
-from .database import Base
+
+class Base(DeclarativeBase):
+    pass
 
 
-class CaseInsensitiveComparator(Comparator):  # pylint: disable=abstract-method
+class CaseInsensitiveComparator(
+    Comparator
+):  # pylint: disable=abstract-method,too-many-ancestors
     def __eq__(self, other):
         return func.lower(self.__clause_element__()) == func.lower(other)
 
 
 class TimestampMixin:
     @declared_attr
     def create_timestamp(cls):  # pylint: disable=no-self-argument
-        return Column(
-            TIMESTAMP(timezone=True), nullable=False, server_default=func.now()
+        return mapped_column(
+            TIMESTAMP(timezone=True),
+            nullable=False,
+            server_default=func.now(),  # pylint: disable=not-callable
         )
 
     @declared_attr
     def update_timestamp(cls):  # pylint: disable=no-self-argument
-        return Column(
+        return mapped_column(
             TIMESTAMP(timezone=True),
             nullable=False,
-            server_default=func.now(),
+            server_default=func.now(),  # pylint: disable=not-callable
             # server_onupdate would be nice to use here, but it's not supported
             # by Postgres
-            onupdate=func.now(),
+            onupdate=func.now(),  # pylint: disable=not-callable
         )
 
 
 class Email(Base, TimestampMixin):
     __tablename__ = "emails"
     __mapper_args__ = {"eager_defaults": True}
 
-    email_id = Column(UUID(as_uuid=True), primary_key=True)
-    primary_email = Column(String(255), unique=True, nullable=False)
-    basket_token = Column(String(255), unique=True)
-    sfdc_id = Column(String(255), index=True)
-    first_name = Column(String(255))
-    last_name = Column(String(255))
-    mailing_country = Column(String(255))
-    email_format = Column(String(1))
-    email_lang = Column(String(5))
-    double_opt_in = Column(Boolean)
-    has_opted_out_of_email = Column(Boolean)
-    unsubscribe_reason = Column(Text)
+    email_id: Mapped[UUID4] = mapped_column(UUID(as_uuid=True), primary_key=True)
```

Comment:
I changed `Column` -> `mapped_column` for every column in every model, but only did the bare minimum of supplying the UUID4 types. In a future commit, we can supply the rest of the types.